templates/xcassets/swift3.stencil

```diff
@@ -1,14 +1,16 @@
 // Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
 
 {% if catalogs %}
+{% set colorAlias %}{{param.colorAliasName|default:"AssetColorTypeAlias"}}{% endset %}
+{% set imageAlias %}{{param.imageAliasName|default:"Image"}}{% endset %}
```

Comment:
Such a shame that these two aliases don't end up to be similar (but I know, would be breaking because `typealias Image` was already existing before :-/ )